crypto.pl

```diff
@@ -33,7 +33,12 @@
 */
 
 :- module(crypto,
-          [ evp_decrypt/6,              % +CipherText, +Algorithm, +Key, +IV, -PlainText, +Options
+          [ crypto_term_hash/3,         % +Data, -Hash, +Options
```

Comment:
I am now using `crypto_data_hash/3`, since the arguments were also denoted as `Data` in several other predicates, and it can be a string, atom or list of characters.